src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetFiltersMappingPage.tsx

```diff
@@ -396,16 +412,16 @@ export default function ReleaseApiDataSetFiltersMappingPage() {
 
             <h3
               className="govuk-heading-l govuk-!-margin-top-8"
-              id="new-filter-options"
+              id={sectionIds.newFilterOptions}
             >
               New filter options ({totalNewFilterOptions}){' '}
               <Tag colour="grey">No action required</Tag>
             </h3>
 
             {totalNewFilterOptions > 0 && filtersMapping ? (
               <Accordion
-                id="new-filter-options-accordion"
-                testId="new-filter-options-accordion"
+                id={`${sectionIds.newFilterOptions}-accordion`}
```

Comment:
Worth having a `sectionIds.newFilterOptionsAccordion` variable just to avoid the risk of this too getting out of sync?